opennms-webapp/src/main/webapp/admin/sched-outages/index.jsp

```diff
@@ -70,6 +70,7 @@
 			WriteablePollOutagesDao.class);
 
 	NotifdConfigFactory.init(); //Must do this early on - if it fails, then just throw the exception to the web gui
+    NotifdConfigFactory.getInstance().reload();
```

Comment:
is that possible to not call reload here ? (your init reload is removed, and postConstruct did not work properly?)